lib/mojito.js

```diff
@@ -129,6 +130,24 @@ Mojito = (function ()
                 }
             }
 
+            // decision adapter if TEST adapter defined, use it;
+            // else if GLOBAL adapter defined, use that;
+            // else use DEFAULT adapter 
+            if (!this.options.decisionAdapter)
```

Comment:
Just to confirm your the logic above. Is this the test object's `decisionAdapter` or the global `decisionAdapter`?

As you mention, we should prioritise:

1. Test object `decisionAdapter`
2. Global `decisionAdapter`
3. Default to `seededRandom`

Reply:
@kingo55 
Yes, you're right. The ```decisionAdapter``` will be assigned to the test object finally even we defined it with: Mojito.options.decisionAdapter = function() {}

Also, we can define different decisionAdapters for different tests, e.g:

```
Mojito.addTest({
    id: 'w51',
    name:'form-error',
    testScope:"user",
    sampleRate: 1,
    options: {
        debug: true, cookieDuration: 90,
        decisionAdapter: function (test)
        {
            if (test.options.decisionCnt === 0)
            {
                return 0.8;
            }
            return 0.6;
        }
    },
    state: 'live',
    recipes:{
        "0":{
            name: "Original",
        },
        "1":{
            name: "Variation A"
        }
    }
});
```

In this case, w51 test has its own ```decisionAdapter``` even we have the global decisionAdapter defined using Mojito.options.decisionAdapter = function() {}

Reply:
Thanks...